Services/Interfaces/web_interface/models/configuration.py

```diff
@@ -37,6 +37,9 @@
 import octobot.constants as octobot_constants
 
 NAME_KEY = "name"
+SYMBOL_KEY = "symbol"
+ID_KEY = "id"
+EXCLUDED_CURRENCY_SUBNAME = tuple(("X Long", "X Short"))
```

Comment:
There is currencies with this end of name ? :open_mouth:

Reply:
Yes unfortunatelly, and since we very likely don't want them, I think it's better to filter them. ex "RealToken 18900 Mansfield St Detroit MI"

Reply:
:open_mouth: